puppetboard/templates/_macros.html

```diff
@@ -173,26 +89,62 @@
     {% if events['skips'] %}<span class="ui small count label skipped">{{events['skips']}}</span>{% else %}<span class="ui small count label">0</span>{% endif%}
   {% endif %}
 {%- endmacro %}
-{% macro render_pagination(pagination) -%}
-  <div class="ui pagination menu">
-  {% if pagination.has_prev %}
-    <a class="item" href="{{url_for_field('page', 1)}}">&laquo; First</a>
-    <a class="item" href="{{url_for_field('page', pagination.page - 1)}}">Prev</a>
-  {% endif %}
-  {% for page in pagination.iter_pages() %}
-    {% if page %}
-      {% if page != pagination.page %}
-        <a class="item" href="{{url_for_field('page', page)}}">{{page}}</a>
-      {% else %}
-        <a class="active item">{{page}}</a>
-      {% endif %}
-    {% else %}
-      <a class="disabled item">...</a>
-    {% endif %}
-  {% endfor %}
-  {% if pagination.has_next %}
-    <a class="item" href="{{url_for_field('page', pagination.page + 1)}}">Next</a>
-    <a class="item" href="{{url_for_field('page', pagination.pages)}}">Last &raquo;</a>
-  {% endif %}
-  </div>
-{% endmacro %}
+
+{% macro datatable_init(table_html_id, ajax_url, default_length, length_selector, extra_options=None, columns) -%}
```

Comment:
Can set the default values for the `default_length` and `length_selector` parameters to `config.LITTLE_TABLE_COUNT` and `config.TABLE_COUNT_SELECTOR`